src/pages/organisations.mdx

```diff
@@ -17,7 +17,7 @@ Is your organisation using Fastify? That's great!
 
 If you would like your organisation to have its logo featured in this list, here's how you do it! You can submit your organisation, your logo and other few relevant details by following these steps:
 
-1. Fork the [Fastify website repository](https://github.com/fastify/website) on GitHub
+1. Fork the [Fastify website repository](https://github.com/fastify/website-next) on GitHub
```

Comment:
Not sure, I would assume this repo would change the name to `website` once it's done. 
And the old one would keep with another name.

Reply:
current [Fastify website repository](https://github.com/fastify/website) doesnt have [this directory](https://github.com/fastify/website-next/tree/main/static) which that guide is addressing
https://github.com/fastify/website-next/blob/4fa3897759fc82cd1cfbf5f7da459fd75b8c2abb/src/pages/organisations.mdx?plain=1#L21
which gets confusing to current visitors to this website

Reply:
> which gets confusing to current visitors to this website

I think this is the point: maybe we should add a banner on this website that stands "🚧 work in progress".
Right now fastify.io is the source of truth - an we will need to align once again the organizations as well before "going to production" 

But in general, I would reason that `website-next` will be renamed for sure to `website`